presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -110,6 +110,12 @@
     @GuardedBy("root")
     private long hardCpuLimitMillis = Long.MAX_VALUE;
     @GuardedBy("root")
+    private long maxExecutionTimeMillis = Long.MAX_VALUE;
```

Comment:
`perQueryExecutionTimeLimitMillis` - adding the words `perQuery` will clarify that this is a per query limit